bin/Powheg/production/2017/13TeV/VBF_H_NNPDF31_13TeV/VBF_H_NNPDF31_13TeV_template.input

```diff
@@ -29,8 +29,8 @@ renscfact  1d0   ! (default 1d0) ren scale factor: muren  = muref * renscfact
 facscfact  1d0   ! (default 1d0) fac scale factor: mufact = muref * facscfact
 withdamp    1      ! (default 0, do not use) use Born-zero damping factor
 testplots  1      ! (default 0, do not) do NLO and PWHG distributions
-pdfreweight 0       ! PDF reweighting
-storeinfo_rwgt 0    ! store weight information
+pdfreweight 1       ! PDF reweighting
```

Comment:
please crosscheck https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T (bottom of the section)